veascan-web/src/consts/bridges.ts

```diff
@@ -42,22 +42,22 @@ export const bridges: IBridge[] = [
     to: sepolia.id,
     inboxAddress: VeaInboxArbitrumSepolia.address as `0x${string}`,
     inboxEndpoint:
-      "https://api.studio.thegraph.com/query/67213/veascan-inbox-arb-sep-devnet/version/latest",
+      "https://api.studio.thegraph.com/query/85918/veascan-inbox-arb-sep-devnet/v0.8.0",
```

Comment:
Same suggestion about extracting to env vars

Reply:
Yep, will do.